src/store/index.ts

```diff
@@ -0,0 +1,43 @@
+import Vue from 'vue'
+import Vuex from 'vuex'
+import {getMovies} from '@/api/BaseApi'
+
+Vue.use(Vuex)
+
+export interface Movie {
+  id: number;
+  title: string;
+  genre: {
+    id: number;
+    name: string;
+  };
+  poster_url: string;
+  length: number;
+  release_date: string;
+  description: string;
+}
+
+export default new Vuex.Store({
+  state: {
+    movies: [] as Array<Movie>,
```

Comment:
BTW - you can also write `Movie[]` or `string[]` or `number[]` - it's the same thing and a bit shorter 💡

Reply:
Yes, I tried that, but that gave me a lot or errors so it was easier to write `as Array<Movie>` :P